src/VstsSyncMigrator.Core/Execution/MigrationContext/WorkItemMigrationContext.cs

```diff
@@ -562,13 +562,14 @@ private MigrationTools._EngineV1.DataContracts.WorkItemData ReplayRevisions(List
                     WorkItemTypeChange(targetWorkItem, skipToFinalRevisedWorkItemType, finalDestType, revision, currentRevisionWorkItem, destType);
 
                     PopulateWorkItem(currentRevisionWorkItem, targetWorkItem, destType);
+
+                    // Todo: Ensure all field maps use WorkItemData.Fields to apply a correct mapping
                     Engine.FieldMaps.ApplyFieldMappings(currentRevisionWorkItem, targetWorkItem);
 
-                    targetWorkItem.ToWorkItem().Fields["System.ChangedBy"].Value =
-                        currentRevisionWorkItem.ToWorkItem().Revisions[revision.Index].Fields["System.ChangedBy"].Value;
+                    // Todo: Think about an "UpdateChangedBy" flag as this is expensive! (2s/WI instead of 1,5s when writing "Migration")
```

Comment:
We have an UpdateChangedBy flas as:

```
    public bool UpdateCreatedDate { get; set; }
        public bool UpdateCreatedBy { get; set; }
```
On `WorkItemMigrationConfig`

Reply:
This is for created by not changed by...

Reply:
My bad :)